src/components/staking/StakingContainer.js

```diff
@@ -151,9 +151,9 @@ export function StakingContainer({ history, match }) {
     if (!validator) {
         validator = validators.filter(validator => validator.accountId === validatorId)[0]
     }
-    const { totalUnstaked, selectedValidator, totalStaked } = currentAccount
+    const { totalUnstaked, selectedValidator, totalStaked, totalUnclaimed } = currentAccount
 
-    const unableToCalcRewards = currentAccount.accountId === accountId && has2fa && !new BN(totalStaked).isZero()
+    const unableToCalcRewards = false && new BN(totalUnclaimed).isZero()
```

Comment:
@mattlockyer this is always `false`, is it intentional?

Reply:
Oh sorry, I should have cleaned this up.

It's something Patrick implemented and it needs to be deprecated because with indexer we can always calc staking rewards